Makefile

```
@@ -18,7 +18,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.5.0
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.5.4
```

This can now be simplified:

```suggestion
DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0
```